transport/tchannel/handler.go

```diff
@@ -110,7 +111,8 @@ func (h handler) handle(ctx context.Context, call inboundCall) {
 		status = tchannel.ErrCodeTimeout
 	}
 
-	call.Response().SendSystemError(tchannel.NewSystemError(status, err.Error()))
+	// TODO: explicitly ignoring error, can we deal with this?
+	_ = call.Response().SendSystemError(tchannel.NewSystemError(status, err.Error()))
```

Comment:
Same here. What we can do is log the error once we have some sort of logging
set up in the library. Maybe switch the TODO to "log these errors"?